### PR TITLE
Create sensor.solax

### PR DESCRIPTION
**Pull request by CountParadox**
**Description:**

Initial adding of docs

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22579

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.solax.markdown [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+comments: false
+sharing: true
+footer: true
+logo: solax-logo.png
+ha_category: Sensor
+ha_release: 0.91
+ha_iot_class: Local Polling
+---
+
+The `solax` component connects home-assistant to Solax solar power inverters. Solax inverters may be connected to a home Wi-Fi network and expose a REST api. This component retrieves information such as photo voltaic power production, battery levels and power, and how much power is being fed back into the grid.
```
Comment (frenck) [inline source/_components/sensor.solax.markdown]:
```suggestion
The `solax` component connects home-assistant to Solax solar power inverters. Solax inverters may be connected to a home Wi-Fi network and expose a REST API. This component retrieves information such as photovoltaic power production, battery levels and power, and how much power is being fed back into the grid.
```

File: source/_components/sensor.solax.markdown [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## {% linkable_title Configuration %}
+
+To use the Solax sensors in your installation, add the following to your configuration.yaml file:
+
+{% raw %}
```
Comment (frenck) [inline source/_components/sensor.solax.markdown]: No need for raw here
```suggestion
```

File: source/_components/sensor.solax.markdown [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: solax
+    ip_address: IP_ADDRESS
+```
+{% endraw %}
```
Comment (frenck) [inline source/_components/sensor.solax.markdown]:
```suggestion

```

File: source/_components/sensor.solax.markdown [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+{% configuration %}
+
```
Comment (frenck) [inline source/_components/sensor.solax.markdown]: Unneeded empty line
```suggestion
```

File: source/_components/sensor.solax.markdown [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+ip_address:
+  description: The IP address of your Solax system.
+  required: true
+  type: string
+{% endconfiguration %}
+
+
+### {% linkable_title Full configuration sample %}
+
+{% raw %}
```
Comment (frenck) [inline source/_components/sensor.solax.markdown]:
```suggestion
```

File: source/_components/sensor.solax.markdown [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+A full configuration entry would look like the sample below.
+
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: solax
+    ip_address: 192.168.0.3
+```
+{% endraw %}
```
Comment (frenck) [inline source/_components/sensor.solax.markdown]:
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]
Looks good @CountParadox! Some minor tweaks and the docs are done!

One additional note: Please rename the file from `sensor.solax.markdown` to `solax.markdown`.

👍

**Comment by CountParadox** [comment]
Done!

**Comment by frenck** [comment]
Thanks for handling those change so fast @CountParadox! 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — DISMISSED** [review]

**Comment by squishykid** [comment]
@frenck parent has been merged

File: source/_components/solax.markdown [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+title: "Solax Sensor"
+description: "Instructions on how to integrate Solax sensor within Home Assistant."
+date: 2019-03-31 04:20
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: solax-logo.png
+ha_category: Sensor
+ha_release: 0.91
```
Comment (klaasnicolaas) [inline source/_components/solax.markdown]:
```suggestion
ha_release: 0.94
```

File: source/_components/solax.markdown [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+{% endconfiguration %}
+
+
+### {% linkable_title Full configuration sample %}
+
+A full configuration entry would look like the sample below.
+
+```yaml
+# Example configuration.yaml entry
+sensor:
```
Comment (klaasnicolaas) [inline source/_components/solax.markdown]: I think that we can remove this one, it's almost the same as above.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**